src/Symfony/Component/HttpKernel/Bundle/Bundle.php

```diff
@@ -135,6 +135,11 @@ public function registerCommands(Application $application)
     {
     }
 
+    public function getPublicPath(): string
+    {
+        return 'Resources/public';
```

Comment:
IMO that should return the absolute dir to be consistent with getLogDir etc, 

https://github.com/symfony/symfony/blob/08c80eb3419606fa6f605037d5f6a392187f90c7/src/Symfony/Component/HttpKernel/Kernel.php#L346

Reply:
@Tobion here we're not in the project context, but in the bundle one. Anyway, bundle always returned such path, this is not changing anything